lib/crewai-tools/src/crewai_tools/tools/truthbear_gauge_tool/Examples.md

```
@@ -0,0 +1,137 @@
+# 🐻 Truth Bear GAUGE Tools — Usage Examples
+
+### 🔧 Tool Initialization
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)